core/domain/exp_services.py

```diff
@@ -1148,17 +1462,34 @@ def _exp_rights_to_search_dict(rights):
 
 
 def _should_index(exp):
+    """
```

Comment:
Returns whether the given exploration should be indexed for future search queries.